components/UserSavedCard.vue

```diff
@@ -0,0 +1,73 @@
+<template>
+  <div>
+    <div v-if="isDefaultCard()" class="is-primary">Primary</div>
+    <div class="card-list">
+      <div class="card-list__left">
+        <div>Icon</div>
+      </div>
+      <div class="card-list__right">
+        <div>Ending :{{ paymentMethod.endingDigit }}</div>
+        <div>Exp : {{ paymentMethod.expiry }}</div>
```

Comment:
I think we should use localization for Primary, Icon, Ending and Exp keywords instead of hardcoding it.